org.eclipse.xtext.xtext.generator/xtend-gen/org/eclipse/xtext/xtext/generator/XtextGeneratorTemplates.java

```diff
@@ -47,6 +47,8 @@
 import org.eclipse.xtext.xtext.generator.model.annotations.IClassAnnotation;
 import org.eclipse.xtext.xtext.generator.model.annotations.SuppressWarningsAnnotation;
 import org.eclipse.xtext.xtext.generator.model.project.IXtextProjectConfig;
+import org.osgi.framework.Bundle;
```

Comment:
should we also check others in the class like org.osgi.framework.BundleContext

Reply:
ping @kthoms

Reply:
OK, I'll update this.

Reply:
done